rllib/agents/ppo/ppo_torch_policy.py

```diff
@@ -223,6 +224,14 @@ def training_view_requirements_fn(policy):
         SampleBatch.NEXT_OBS: ViewRequirement(SampleBatch.OBS, shift=1),
         # VF preds are needed for the loss.
         SampleBatch.VF_PREDS: ViewRequirement(shift=0),
+        # Needed for postprocessing.
+        SampleBatch.ACTION_DIST_INPUTS: ViewRequirement(shift=0),
+        SampleBatch.ACTION_LOGP: ViewRequirement(shift=0),
+        # Created during postprocessing.
+        Postprocessing.ADVANTAGES: ViewRequirement(
+            shift=0, created_during_postprocessing=True),
```

Comment:
Why do we need this created_during_postprocessing flag, is it necessary?

Reply:
It's more for safety reasons: When we generate the sample batch for postprocessing, some column in the view_reqs (we use the same as for training) may not exist yet (b/c these are only added during postprocessing). This lets the SampleCollector know, that it's ok that these columns are still missing in the main-buffer.
True: For the developer, it may be simpler to not require this flag. Therefore: removed it and made it skip columns that do not exist yet (in the postprocessing sample batch generating code, not in the train batch generating code, where a missing col will be an error).